### PR TITLE
Add Load and Save

### PR DESCRIPTION
**Pull request by modakshantanu**

### CODE REVIEW

**Comment by modakshantanu** [comment]
image

**Comment by modakshantanu** [comment]
Manual Travis check if the online one fails

**Comment by modakshantanu** [comment]
I have commented out Some of InvalidComandTest

File: build.gradle [hunk 1]
```diff
@@ -77,12 +77,12 @@ test {
     testLogging {
         events "passed", "skipped", "failed"
     }
-    systemProperty 'java.awt.headless', 'true'
-    systemProperty 'testfx.robot', 'glass'
-    systemProperty 'testfx.headless', 'true'
-    systemProperty 'prism.order', 'sw'
-    systemProperty 'prism.text', 't2k'
-    systemProperty 'testfx.setup.timeout', '2500'
+//    systemProperty 'java.awt.headless', 'true'
```
Comment (modakshantanu) [inline build.gradle]: oops how did this get through

Comment (nwjbrandon) [reply build.gradle]: Yup, please uncomment this

File: src/main/java/com/algosenpai/app/MainApp.java [hunk 2]
```diff
@@ -78,7 +92,7 @@ private void startMain(Stage stage) throws IOException {
         AnchorPane ap = fxmlLoader.load();
         Scene scene = new Scene(ap, MAINWINDOW_WIDTH, MAINWINDOW_HEIGHT);
         stage.setScene(scene);
-        fxmlLoader.<Ui>getController().setLogic(logic, stats);
+        fxmlLoader.<Ui>getController().setLogic(logic, stats, wasDatafileCorrupted);
```
Comment (nwjbrandon) [inline src/main/java/com/algosenpai/app/MainApp.java]: Prefer if you named this as isDataFileCorrupted

File: src/main/java/com/algosenpai/app/storage/Storage.java [hunk 3]
```diff
@@ -16,28 +17,19 @@
...
-            return newUserStats.toString();
+            throw new FileNotFoundException();
         }
 
         try {
             return Files.readString(Paths.get(filePath), StandardCharsets.US_ASCII);
         } catch (IOException e) {
             e.printStackTrace();
         }
+
```
Comment (modakshantanu) [inline src/main/java/com/algosenpai/app/storage/Storage.java]: I would like to apologize for what seems like a blatant attempt to increase my line count at the expense of readability. I hope you understand this was an accident

File: src/test/java/com/algosenpai/app/commands/VolumeCommandTest.java [hunk 4]
```diff
@@ -40,7 +40,7 @@ void testSoundLevelWrongInput() throws IOException {
     }
 
     @Test
-    void testSoundLevelMissingInput() throws IOException {
+    void testSoundLevelMissingInput() throws Exception {
```
Comment (nwjbrandon) [inline src/test/java/com/algosenpai/app/commands/VolumeCommandTest.java]: Is there a reason for change from Exception to IOException?

File: src/main/java/com/algosenpai/app/storage/Storage.java (hunk 3 above)
Comment (nwjbrandon) [reply src/main/java/com/algosenpai/app/storage/Storage.java]: I believe Reposense will not detect this extra line

File: src/test/java/com/algosenpai/app/commands/VolumeCommandTest.java (hunk 4 above)
Comment (modakshantanu) [reply src/test/java/com/algosenpai/app/commands/VolumeCommandTest.java]: Because now there are 2 different exceptions throws so I was lazy to put both

Comment (nwjbrandon) [reply src/test/java/com/algosenpai/app/commands/VolumeCommandTest.java]: Okay, I believe it is not a huge violation if it is a test case.

File: build.gradle (hunk 1 above)
Comment (modakshantanu) [reply build.gradle]: Its updated now

**Comment by modakshantanu** [comment]
image
Get Rekt

**Pull request merged**